source/dub/dub.d

```diff
@@ -105,28 +107,50 @@ class Dub {
 		This property can be altered, so that packages which are downloaded as part
 		of the normal upgrade process are stored in a certain location. This is
 		how the "--local" and "--system" command line switches operate.
-	*/
-	PlacementLocation defaultPlacementLocation = PlacementLocation.user;
 
+		If it is relative, it is resolved relative to the current working directory
+	*/
+	Path defaultRepoPath() @property
+	{
+		return m_defaultRepoPath;
+	}
+	/// Ditto
+	void defaultRepoPath(Path path) @property
+	{
+		path = makeAbsoluteRelativeToCwd(path);
+		//assert(m_packageManager.isManagedPath(path, false), path.toNativeString);
+		m_defaultRepoPath = path;
+	}
```

Comment:
`defaultRepoPath` fulfils exactly the same purpose as `defaultPlacementPath`, but is based on paths instead of `PlacementLocation`